src/api/db/seeds.rb

```diff
@@ -174,6 +174,9 @@
 at = ans.attrib_types.where(name: 'RejectBranch').first_or_create(value_count: 1)
 at.attrib_type_modifiable_bies.where(role_id: maintainer_role.id).first_or_create
 
+at = ans.attrib_types.where(name: 'LimitReleaseSourceProject').first_or_create
+# at.attrib_type_modifiable_bies.where(role_id: maintainer_role.id).first_or_create
```

Comment:
In the original PR we missed setting the roles of this attribute type. I'm not sure what roles add here... `maintainer`, `admin`, ...

Reply:
maintainer

Reply:
Ok. Done.